src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -247,20 +247,22 @@
                      {chain-id           :chain-id
                       network-value-type :type
                       total-amount       :total-amount}]
-                  (let [status (cond (and (= network-value-type :not-available)
-                                          loading-routes?
-                                          token-not-supported-in-receiver-networks?)
-                                     :loading
-                                     (= network-value-type :not-available)
-                                     :disabled
-                                     :else network-value-type)]
+                  (let [status           (cond (and (= network-value-type :not-available)
+                                                    loading-routes?
+                                                    token-not-supported-in-receiver-networks?)
+                                               :loading
+                                               (= network-value-type :not-available)
+                                               :disabled
+                                               :else network-value-type)
+                        amount-formatted (-> (rf/sub [:wallet/send-amount-fixed total-amount])
```

Comment:
`rf/sub` like this feels odd. do we compute and `sub` like this at other places ?

afaik, `rf/sub` is similar to `useState` hook, so it should be called at the top level. Derivative values should be computed separately.

Reply:
`rf/sub` just derefs the atom returned by `rf/subscribe`, so there's no hooks anywhere. I've seen it used in a few places, but not sure if  `(-> ...)` results in a different scope when compiled to js.

```clj
(def sub (comp deref re-frame/subscribe))
```

Reply:
The subscribe returns a ratom, as long as it's used in a reactive context, everything should be fine 👍